src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -119,7 +120,7 @@ export default class AnnotationColumnFormatter
         return true;
     }
 
-    public static getIndicatorData(copyNumberData:DiscreteCopyNumberData[], oncoKbData:IOncoKbData): IndicatorQueryResp|undefined
+    public static getIndicatorData(copyNumberData:DiscreteCopyNumberData[], oncoKbData:IOncoKbData, studyIdToStudy?: {[studyId:string]:CancerStudy}): IndicatorQueryResp|undefined
```

Comment:
should studyIdToStudy be optional?

Reply:
Yes, this should be the optional since when we sort and download the data, there is no need to consider the studyIdToStudy.